test/gw-ci/CMakeLists.txt

```diff
@@ -1,11 +1,41 @@
+# Function that adds a test for a given GW task
+function(add_task task_name test_prefix cycle pslot RUNTESTS PROJECT_SOURCE_DIR)
```

Comment:
thoughts on adding `run` also?

Reply:
@CoryMartin-NOAA What do you mean?

Reply:
`gdas` or `enkfgdas` or `gfs`

Reply:
in the workflow the full jobs are defined as `$run$task` so the task name is just `fcst` but the job would be `gdasfcst`

Reply:
I think that would probably add unnecessary complexity. The ```add_task``` function is meant to split metatasks into subtasks, which doesn't really matter for ```gfs```, because I think none of the  ```gfs```  tasks are indeed metatasks. If you want to add ```gfs``` tasks for example,  you would just add ```gfs_prep```, ```gfs_anal```, etc to the ```FULL_CYCLE_TASKS``` list

Reply:
And ```gdas``` and ```enkfgdas``` metatasks are very different

Reply:
ha, good idea @CoryMartin-NOAA . We keep breaking the gfs cycle, might be good to test every once in a while.